bridge/client/webrtc.js

```diff
@@ -54,7 +54,7 @@
         "video": [
             { "encodingName": "H264", "type": 103, "clockRate": 90000,
                 "ccmfir": true, "nackpli": true, "ericscream": true, /* "nack": true, */
-                "parameters": { "profileLevelId": "42e01f", "levelAsymmetryAllowed": 1, "packetizationMode": 1 } },
+                "parameters": { "levelAsymmetryAllowed": 1, "packetizationMode": 1 } },
```

Comment:
I was not aware that this had been added (I recommended not adding it in the PR). Anyway, then I'll make sure it is added back.